Index: src/constants/routes.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>export const SIGN_UP = '/signup';\r\nexport const SIGN_IN = '/signin';\r\nexport const LANDING = '/';\r\nexport const HOME = '/home';\r\nexport const ACCOUNT = '/account';\r\nexport const ADMIN = '/admin';\r\nexport const PASSWORD_FORGET = '/pw-forget';\r\nexport const ABOUT_PAGE = '/About';\r\nexport const HOSPITAL_ADDRESS = '/Hospital';\r\n\r\nexport const DONOR_HOME = '/donor'\r\nexport const HOSPITAL_HOME = '/hospital'\r\nexport const CONTACT_US ='/index.js';\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/constants/routes.js	(revision be60224434811f7b390fd43008e7155bef49bc0a)
+++ src/constants/routes.js	(date 1594349329245)
@@ -10,4 +10,4 @@
 
 export const DONOR_HOME = '/donor'
 export const HOSPITAL_HOME = '/hospital'
-export const CONTACT_US ='/index.js';
+
